### DISCUSSION TITLE
The errors occurred after installing ApprovalBundle plugin.

### DISCUSSION THREAD

**Discussion by pylearn2030** [kimai/kimai#4706] [Plugins]
Dear,
I upgraded Kimai 1.30.0 to 2.5.0 successfully.
Currently, I got an error occurred while installing ApprovalBundle plugin.
++++++++++++++++++++++++++++++++++++++++++++++++++++++
CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'kimai_database.kimai2_ext_approval_overtime_history' doesn't exist" at /var/www/html/kimai/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 49 {"exception":"[object] (Doctrine\DBAL\Exception\TableNotFoundException(code: 1146): An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'kimai_database.kimai2_ext_approval_overtime_history' doesn't exist at /var/www/html/kimai/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:49)\n[previous exception] [object] (Doctrine\DBAL\Driver\PDO\Exception(code: 1146): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'kimai_database.kimai2_ext_approval_overtime_history' doesn't exist at /var/www/html/kimai/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'kimai_database.kimai2_ext_approval_overtime_history' doesn't exist at /var/www/html/kimai/vendor/doctrine/dbal/src/Driver/PDO/Statement.php:130)"} []
++++++++++++++++++++++++++++++++++++++++++
It will occur the error "500 Something is wrong" when accessing to "Hours approval"
![image](https://github.com/kimai/kimai/assets/162970815/26f09a62-b402-435d-89cf-a2cee1f4bc38)

Thank you for your help!

**Comment by kevinpapst** [comment]
> Currently, I got an error occurred while installing ApprovalBundle plugin.

And why do you post here?

**Reply by pylearn2030** [reply]
Hi Kevin,
You guided me to open this topic on discussion forum.
![image](https://github.com/kimai/kimai/assets/162970815/766636c3-4dc2-4e95-adfd-19ecf99dbcef)
Where can I post this topic?
Thank you!
